packages/react-core/src/components/Dropdown/Dropdown.tsx

```diff
@@ -114,20 +114,18 @@ const DropdownBase: React.FunctionComponent<DropdownProps> = ({
       ) {
         if (onOpenChangeKeys.includes(event.key)) {
           onOpenChange(false);
-          toggleRef.current?.focus();
+          toggleRef.current?.focus({preventScroll:true});
         }
       }
     };
 
     const handleClick = (event: MouseEvent) => {
       // toggle was opened, focus on first menu item
       if (isOpen && shouldFocusFirstItemOnOpen && toggleRef.current?.contains(event.target as Node)) {
-        setTimeout(() => {
```

Comment:
What problem was this attempting to solve, as I was able to use the component without this, with an apparent performance boost (of 10ms). 
In the case highlighted, using loading state, I'm unsure how the 10ms wait time could possibly assist in focusing, as the dependency on the API would likely be much longer than that.

Reply:
Originally the 10ms timeout was attempting to resolve the scroll jump bug, which was confirmed to have worked in at least 1 product. Outside of that, keeping the timeout for a load state might be more advantageous if we implement a prop to control the timeout rather than hardcoding it to 10ms (currently tracked by issue https://github.com/patternfly/patternfly-react/issues/11020).

@tlabaj would we still want to implement a prop for controlling the timeout duration, or do we want to completely remove the setTimeout and close out 11020 (if a consumer wants a load state sort of implementation, do we just point to our custom menu examples that would require them to wire something up)? Maybe depends on whether we can get confirmation from other products that this new approach resolves the issue for them.

Reply:
I can re-add the timeout with a prop for the timing.

Reply:
Done!